nncf/torch/quantization/layers.py

```diff
@@ -90,11 +90,40 @@ def from_config(cls, qconfig: QuantizerConfig, narrow_range: bool,
                    is_quantized_on_export,
                    compression_lr_multiplier)
 
+    def get_save_parameters(self):
+        return PTQuantizerSpecParameters(self).get_state()
```

Comment:
why do you need a subset of params?
why not save the spec itself?

Reply:
Part of the info from  `PTQuantizerSpec` is stored in `SingleConfigQuantizerSetup` so there is no need to repeat it.
`PTQuantizerSpecParameters` is not sufficient to restore `PTQuantizerSpec` but joint info from  `SingleConfigQuantizerSetup` and `PTQuantizerSpecParameters` is. So I suggest to leave `PTQuantizerSpecParameters` as it is for saving to CompressionState purposes but to restore `PTQuantizerSpec` from joint info. What do you think?

Also `scale_shape` needs to be added to `PTQuantizerSpecParameters` to restore `PTQuantizerSpec`

Reply:
don't like these dependencies for restoring different objects.

quantizer_spec should be restored from quantizer_spec's state and nothing else. 
if you need some extra parameters, keep and restore them outside of spec class.